app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/ui/main/MainViewModel.kt

```diff
@@ -16,7 +16,8 @@ class MainViewModel : ViewModel() {
     internal fun fetchFurnitures(furnitureName: String) {
         _furnitures = furnitureService.fetchFurnitures(furnitureName)
     }
+
     internal var furnitures: MutableLiveData<ArrayList<Furniture>>
         get() {return _furnitures}
-        set(value: MutableLiveData<java.util.ArrayList<Furniture>>) {_furnitures.value}
+        set(value) {_furnitures = value}
```

Comment:
Update setter to correctly set the value.

Reply:
Good.